Sources/EventBus.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+class EventBus<EventName: Hashable> {
```

Comment:
I think we can just use `NotificationCenter.default`, instead of inventing a new event bus.